src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const thead = table.querySelector('thead');
+const tbody = table.querySelector('tbody');
+const tfoot = table.querySelector('tfoot');
+const thElements = document.querySelectorAll('table thead th');
+const positionTh = thElements[1];
```

Comment:
Make sure that the second 'th' element exists in the 'thead'. If the table structure changes or if there are fewer columns, this could cause an 'undefined' error.